src/main/java/com/hazelcast/cloud/Client.java

```diff
@@ -71,44 +69,41 @@ private static void mapExample(HazelcastInstance client) {
      * @param client - a {@link HazelcastInstance} client.
      */
     private static void sqlExample(HazelcastInstance client) {
-
         System.out.println("Creating a mapping...");
-        // See: https://docs.hazelcast.com/hazelcast/5.0/sql/mapping-to-maps
+        // See: https://docs.hazelcast.com/hazelcast/5.1/sql/mapping-to-maps
         final String mappingQuery = ""
-            + "CREATE OR REPLACE MAPPING cities TYPE IMap"
-            + " OPTIONS ("
-            + "     'keyFormat' = 'java',"
-            + "     'keyJavaClass' = 'java.lang.String',"
-            + "     'valueFormat' = 'java',"
-            + "     'valueJavaClass' = 'java.lang.String'"
-            + " )";
+                + "CREATE OR REPLACE MAPPING capitals TYPE IMap"
+                + " OPTIONS ("
+                + "     'keyFormat' = 'varchar',"
+                + "     'valueFormat' = 'varchar'"
+                + " )";
         try (SqlResult ignored = client.getSql().execute(mappingQuery)) {
```

Comment:
```suggestion
        SqlService sqlService = client.getSql();
        try (SqlResult ignored = sqlService.execute(mappingQuery)) {
```

Also reuse the same for other calls.

Reply:
can't do that after doing your other suggestion about splitting the sample into smaller pieces

Reply:
I think that you can still do that by just passing the `SqlService` as the parameter to the methods.

Reply:
OK, I did that